templates/user/settings/organization.tmpl

```diff
@@ -0,0 +1,33 @@
+{{template "base/head" .}}
+<div class="user settings account_link">
+	{{template "user/settings/navbar" .}}
+	<div class="ui container">
+		{{template "base/alert" .}}
+		<h4 class="ui top attached header">
+			{{.i18n.Tr "settings.orgs"}}
+			<div class="ui right">
```

Comment:
I think there should be `if` to check if user has rights to create organizations